### DISCUSSION TITLE
How to cite KaTeX?

### DISCUSSION THREAD

**Discussion by sthoch** [KaTeX/KaTeX#4208] [Q&A] [answered]
Dear all,
our project uses KaTeX to display math. In a technical report, I wish to give credit where it's due and cite the amazing KaTeX project. However, I am unsure how to do so - who should I put as the authors, etc. Is there a publication one should use? Should one cite the rep, or the page?
Thanks in advance.

**Comment by k4b7** [comment, marked as answer]
@sthoch I think linking to the GitHub repo should be sufficient.

**Comment by edemaine** [comment]
@sthoch Thanks for asking!  Sometimes, when citing projects like this, I just use a footnote, as in:
```latex
...using KaTeX\footnote{\url{https://github.com/KaTeX/KaTeX/}}...
```
If you want to use BibTeX, though, there isn't a clear author list (or a very short one), so I'd probably just leave it blank, as in:
```bibtex
@misc{KaTeX,
  title = {KaTeX},
  howpublished = {\url{https://github.com/KaTeX/KaTeX/}},
}
```

**Comment by k4b7** [comment]
We should probably add this to the docs.

**Comment by ylemkimon** [comment]
Maybe we should publish a paper 😄

**Comment by sven-frotscher** [comment]
There is a way to give any repo a DOI: https://guides.github.com/activities/citable-code/

**Comment by k4b7** [comment]
@sven-frotscher can DOI's be referenced in papers?  It seems like an extra level of indirection.  Is the problem being solved that the software being cited may be moved or removed from the internet?

**Comment by sven-frotscher** [comment]
IIUC, a DOI is like a URN (or permalink), potentially redirecting to whatever online location is current.
Zenodo (the service mentioned in the previously linked activity) apparently archives every release at CERN which seems quite stable.
